File: Iniciante/URI 1060.cs
using System;

class URI
{

  public static void Main(string[] args)
  {

    double[] num= new double[6];
    int val = 0; int i;

    for (i = 0; i <= 5; i++)
    {
      num[i] = Double.Parse(Console.ReadLine());
      if (num[i] > 0)
      {
        val++;
      }
    }

    Console.WriteLine(val + " valores positivos");

  }
}

File: Iniciante/URI 1060.c
#include <stdio.h>

int main()
{

  float num[6];
  int val = 0;
  int i;

  for (i = 0; i <= 5; i++)
  {
    scanf("%f", &num[i]);
    if (num[i] > 0)
    {
      val++;
    }
  }

  printf("%d valores positivos\n", val);

  return 0;
}